### PR TITLE
README: Provide the new arg to NewExporter

### PR DESCRIPTION
**Pull request by murrayc-egym**
By just providing nil for the map of optional custom tags.

(Those view.Register() calls should also have error-checking, by the way.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,10 @@ func main() {
     defer influxClient.Close()
 
     // Register our custom exporter to opencensus
-    view.RegisterExporter(influxdb.NewExporter(influxClient, influxDB, func(err error) {
-        log.Fatal(err)
-    }))
+    exporter := egymInfluxExporter.NewExporter(client, config.Monitoring.InfluxDatabase, func(err error) {
+        logrus.Errorf("error while registering Opencensus exporter: %v", err)
+    }, nil)
+    ocView.RegisterExporter(exporter)
```
Comment (mwuertinger) [inline README.md]: `ocView` is undefined

File: README.md [hunk 2]
```diff
@@ -36,9 +36,10 @@ func main() {
     defer influxClient.Close()
 
     // Register our custom exporter to opencensus
-    view.RegisterExporter(influxdb.NewExporter(influxClient, influxDB, func(err error) {
-        log.Fatal(err)
-    }))
+    exporter := egymInfluxExporter.NewExporter(client, config.Monitoring.InfluxDatabase, func(err error) {
+        logrus.Errorf("error while registering Opencensus exporter: %v", err)
```
Comment (mwuertinger) [inline README.md]: `logrus` is not imported and i think we should stick to the standard library in examples as not everybody might know `logrus`.

File: README.md [hunk 3]
```diff
@@ -36,9 +36,10 @@ func main() {
     defer influxClient.Close()
 
     // Register our custom exporter to opencensus
-    view.RegisterExporter(influxdb.NewExporter(influxClient, influxDB, func(err error) {
-        log.Fatal(err)
-    }))
+    exporter := egymInfluxExporter.NewExporter(client, config.Monitoring.InfluxDatabase, func(err error) {
```
Comment (mwuertinger) [inline README.md]: - it's called `influxClient`, not `client`
- where does `config` come from?

Comment (murrayc-egym) [reply README.md]: Thanks. This whole commit was particularly inept of me. I've (force) pushed an improved commit to this PR.

File: README.md [hunk 4]
```diff
@@ -31,14 +31,15 @@ func main() {
         Password: influxPass,
     })
     if err != nil {
-        log.Fatal("InfluxDB client connection failure: ", err)
+        log.Fatalf("InfluxDB client connection failure: %v", err)
```
Comment (mwuertinger) [inline README.md]: Why change this?

Comment (murrayc-egym) [reply README.md]: I don't know if log.Fatal() does what we hope that it does. It's note well documented:
https://golang.org/src/log/log.go?s=9291:9319#L295

But if you are sure about it, I can happily revert that change.

Comment (mwuertinger) [reply README.md]: "Fatal is equivalent to Print() ..."
"Print calls Output to print to the standard logger. Arguments are handled in the manner of fmt.Print."
"Print formats using the default formats for its operands and writes to standard output. Spaces are added between operands when neither is a string."

Sounds pretty unambiguous to me but we can also just use `Fatalf` as the end result is the same.

**Pull request merged**